cypress/plugins/index.ts

```diff
@@ -1,6 +1,3 @@
-/* eslint-disable import/no-extraneous-dependencies */
-/* eslint-disable global-require */
-
 /// <reference types="cypress" />
 // ***********************************************************
 // This example plugins/index.js can be used to load plugins
```

Comment:
I think you can remove this file? I'm not sure

Reply:
I am not sure, I will leave it just in case